src/components/activities/adminActivity/AdminActivityContent/AdminActivityContent.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { PropsWithChildren, ReactNode } from "react";
+import classes from "./AdminActivityContent.module.scss";
+
+type TProps = PropsWithChildren<{
+  title: ReactNode;
+}>;
```

Comment:
```suggestion
type TProps = {
  title: ReactNode;
  children: ReactNode;
};
```

There is this weird thing about `PropsWithChildren` that makes it [ambiguous](https://dev.to/maafaishal/unsafe-propswithchildren-utility-type-in-react-typescript-app-3bd3). I would recommend we don't use it in the codebase because it can lead to unpredicted behaviour. When you don’t specifically need to enforce or interact with the children within the component, using the standard `children: React.ReactNode;` is sufficient. ([source](https://medium.com/@colorsong.nabi/propswithchildren-vs-reactnode-in-typescript-c3182cbf7124))